server/apm/apm-server/apm-engine/src/main/resources/metrics/metrics.json

```diff
@@ -1,257 +1,279 @@
 [
   {
     "name": "apm_service_cpm",
-    "index": "segment",
+    "index": "holoinsight-span",
```

Comment:
This line changes the index name.
Does it have any bad influence on upgrading HoloInsight from old version to new version? (compatibility)

Reply:
Don't worry, this change is just to remove the original obsolete file `metric.json`, and then rename the production-available file `metric-span.json` to `metric.json`. This change has been verified in my local environment.